pages/common/fls.md

```diff
@@ -0,0 +1,16 @@
+# fls
+
+> List files and directories in an image file or device.
+> More information at https://wiki.sleuthkit.org/index.php?title=Fls.
+
+- Built a recursive fls list over a device:
+
+`fls -r -m {{Original_Path}} {{/dev/examplepartition}}`
```

Comment:
```suggestion
`fls -r -m {{original_path}} {{/dev/examplepartition}}`
```

Thank you for correct one in 510f0b0. If you could fix this one too, it would be awesome.